subsys/net/ip/Kconfig

```diff
@@ -685,12 +685,14 @@ config NET_CONTEXT_DSCP_ECN
 
```

Comment:
Please do not add commit that "fixes" the earlier commits in this PR. So place the review changes to corresponding earlier commits. Having a separate commit for fixes/changes is nice for the reviewer but it hurts bisectability so we avoid doing it.